posts/2022-12-20-automation-star-conference.adoc

```diff
@@ -0,0 +1,142 @@
+= AutomationSTAR 2022 Conference
+TehreemFatima07
+v1.0, 2022-12-06
+:title: AutomationSTAR 2022 Conference
+:imagesdir: ../media/2022-12-20-automation-star-conference
+:lang: en
+:tags: [conference, automation testing]
+
+
+October 17 and 18 were the days for the Europe’s largest testing conference, https://automation.eurostarsoftwaretesting.com/[Automation STAR 2022]. About 100 passionate software testers gathered together in https://www.hilton.com/en/hotels/muchitw-hilton-munich-park/[Hilton Munich Park], for making automation testing experiences better.
+
+++++
+</br>
+++++
+
+Conference was focused mainly on test automation which covered a variety of different topics such as event driven test automation, cypress, test automation frameworks, strategy, functional testing, codeless test automation and soft skills of automation.
+
+== Things before the Conference
+
+One week before the conference an email was sent to all the attendees which has the instructions about the conference. The email also contained the link for the https://whova.com/portal/autom1_202210/?source=download_page/[whova app] where attendees can get live updates and chat with others. After registering on the app, all the information about the sessions and the speakers were available. Attendees can also initiate the discussion and can contact others like the ice breaking sessions. One interesting thing about the app was the Photo competition, where participants can post the conference pictures and picture with the maximum likes will receive the price.
```

Comment:
Nit: Since you haven't mention the price before I'd recommend to change `the price` to `a price`